Uranus

```diff
@@ -1 +1,5 @@
-Uranus 27 moons: Titania Oberon Umbriel Ariel Miranda
+Uranus 27 moons:
+Cordelia, Ophelia, Bianca, Cressida, Desdemona, Juliet, Portia, Rosalind,
+Cupid, Belinda, Perdita, Puck, Mab, Miranda, Ariel, Umbriel, Titania and
+Oberon, Francisco, Caliban, Stephano, Trinculo, Sycorax, Margaret, Prospero,
+Setebos and Ferdinand.
```

Comment:
We want to avoid ambiguity whenever possible:
https://en.wikipedia.org/wiki/Serial_comma#Unresolved_ambiguity